pretix_taler/static/pretix_taler/check.js

```diff
@@ -0,0 +1,16 @@
+$(function () {
+    var check = function () {
```

Comment:
Fürs debugging ist es meines Wissens einfacher, wenn die Funktion einen Namen hat. Bei diesem Code-Konstrukt wird IMHO der Variable check eine anonyme Funktion zugewiesen. Man könnte hier also schreiben:

```suggestion
    function check () {
```

Ist aber nur theoretischer Natur, glaube ich.